File: yt/yt/flow/examples/java/retryable_async_request/retryable_async_request/src/main/java/tech/ytsaurus/flow/examples/retryableasyncrequest/NodeCompanionMain.java
package tech.ytsaurus.flow.examples.retryableasyncrequest;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication
public class NodeCompanionMain {

    private NodeCompanionMain() {
    }

    // [BEGIN main]
    public static void main(String[] args) throws Exception {
        new SpringApplicationBuilder(NodeCompanionMain.class).run(args);
    }
    // [END main]
}

File: yt/yt/flow/examples/kotlin/url_downloader/url_downloader/src/main/kotlin/tech/ytsaurus/flow/examples/urldownloader/NodeCompanionMain.kt
package tech.ytsaurus.flow.examples.urldownloader

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder

@SpringBootApplication
open class NodeCompanionMain {
    companion object {
        // [BEGIN main]
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplicationBuilder(NodeCompanionMain::class.java)
                .run(*args)
        }
        // [END main]
    }
}
